proto/anki/cards.proto

```diff
@@ -40,8 +40,8 @@ message Card {
   uint32 remaining_steps = 14;
   sint32 original_due = 15;
   int64 original_deck_id = 16;
-  uint32 flags = 17;
-  string data = 18;
+  uint32 flags = 18;
+  generic.UInt32 original_position = 17;
```

Comment:
The build fails if `original_position` is assigned the highest field number, and unlike with a syntax error there is no specific error message. Do you know why? I couldn't find anything about that in the official docs.

Reply:
Hmm, I don't seem to have a problem using 18 here. Once the other build error is resolved, could you try pushing the original order to CI? If it passes, I can try on a Windows machine.

Reply:
Maybe it's a cache problem. 🤔 I also get this:
![Unbenannt](https://user-images.githubusercontent.com/48286681/154961887-379b10a2-7bf2-49a2-afa2-38e4bc071565.jpg)

Building works fine otherwise, and `bazel clean` didn't help, though. I'll look into it further.

Reply:
Rust Analyzer may need you to ctrl+shift+p: rust analyzer: run, then run a cargo check, in order for the build script to get re-run.

Reply:
Main builds just fine... All right then. Keep your secrets, protobuf.